funnel/views/proposal.py

```diff
@@ -184,9 +184,9 @@ def proposal_edit(profile, space, proposal):
 def proposal_status(profile, space, proposal):
     form = ProposalStatusForm()
     if form.validate_on_submit():
-        proposal.status = form.status.data
+        proposal._status = form.status.data
         db.session.commit()
-        flash(_("The proposal has been ") + PROPOSALSTATUS[proposal.status].lower(), 'success')
+        flash(_("The proposal has been ") + PROPOSALSTATUS[proposal.status.value].lower(), 'success')
```

Comment:
`flash(_("The proposal's status has changed to ") + proposal.state.label.title, 'success')`

However, this entire view needs to go away and be replaced with transitions (separate PR).